index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <section class="tool-box">
+        <div class="headder">
+            <h4 class="name">Welcome</h4>
+            <img class="icon" src="images/Union.png" alt="icon">
+        </div>
+    </section>
+    <!-- welcome_page -->
+    <section class="welcome_page">
+        <!-- welcome message -->
+        <div class="welcome_message">
+            <h1 class="greeting">Hey there. I'm Bobby</h1>
+            <h2 class="work">I am a software developer</h2>
+            <h3 class="service">
+                I can help you build a product , feature or website Look through some 
+                of my work and experience! If you like what you see and have a project 
+                you need coded, don’t hestiate to contact me.
+            </h3>
+        </div>
+        <!-- end of welcome message -->
+        <!-- social media -->
+        <ul class="social_media">
+            <li>
+                <a href="https://github.com/Victor-chiemerie">
+                    <img src="images/github.png" alt="github">
+                </a>
+            </li>
+            <li>
+                <a href="https://www.linkedin.com/in/victor-chiemerie-460475246/">
+                    <img src="images/linkedin.png" alt="linkedin">
+                </a>
+            </li>
+            <li>
+                <a href="https://twitter.com/VictorChiemer20">
+                    <img src="images/Twitter.png" alt="twitter">
+                </a>
+            </li>
+        </ul>
```

Comment:
[ Required ] Please align this social media list as illustrated in the Figma template, please see the screenshot below for more details on this. The white space marked in red should not be there, you can implement this using the `CSS` below.
```CSS
  .welcome_page ul {
    padding-top: 40px;
    display: flex;
    height: 22px;
    align-content: space-between;
}
```
## Figma template
<img src="https://user-images.githubusercontent.com/16392046/198832200-90644adc-6fbd-494a-a17a-5ee59233a07d.png" width="300px" />

## Current implementation
<img src="https://user-images.githubusercontent.com/16392046/198832317-2953995b-2328-4688-a854-2c99174144ab.png" width="300px" />